examples/step-76/step-76.cc

```diff
@@ -594,7 +594,7 @@ namespace Euler_DG
 
 
   // The following function does an entire stage of a Runge--Kutta update and is
-  // - alongside the slightly modified setup - the heart of this tutorial
+  // alongside the slightly modified setup - the heart of this tutorial
```

Comment:
This fixes the problem that doxygen formats this as a list, but I think the grammar is a bit awkward now. I would do 
" ... and is, alongside the slightly modified setup, the ..."